website/content/en/docs/building-operators/golang/migration.md

```diff
@@ -230,11 +230,11 @@ test: generate fmt vet manifests
 
 ## Migrate your tests
 
-For the new layout, you will see that `controllers/suite_test.go` is created. This file contains boilerplate for executing integration tests using [envtest][envtest] with [ginkgo](https://onsi.github.io/ginkgo/) and [gomega](https://onsi.github.io/gomega/).
+For the new layout, you will see that `controllers/suite_test.go` is created when a controller is scaffolded by the tool. This file contains boilerplate for executing integration tests using [envtest][envtest] with [ginkgo](https://onsi.github.io/ginkgo/) and [gomega](https://onsi.github.io/gomega/).
```

Comment:
I guess we should just be consistent with how we handle links. Add ginkgo and gomega to the bottom with the other links.